src/components/FieldFunction.tsx

```diff
@@ -370,7 +370,7 @@ export default class FieldFunction extends React.Component<FieldFunctionProps, F
     else if (dataType === "data_function") {
       // TODO: Rename to FieldFunction **this file** shouldn't be called that
       specField = (
-        <DataProperty
+        <DataPropertyInternal
```

Comment:
Is this correct? I wouldn't expect to see this, moreover, I think it should not be exported.